src/converters/notion/index.ts

```diff
@@ -0,0 +1,232 @@
+import { parse } from 'csv-parse/sync';
+import { TanaIntermediateAttribute, TanaIntermediateFile, TanaIntermediateNode } from '../../types/types';
+import { idgenerator } from '../../utils/utils';
+import { IConverter } from '../IConverter';
+
+const isPageLink = (s: string) => {
+    return !!s.match(/\.md$/)
+}
+
+const parseFileContents = (fileContent: string): Record<string, string>[] => {
+    // Looks like notion exporter inserts a Zero Width No-Break Space character at the start of the file.
+  return parse(fileContent.slice(1), {columns: true, skip_empty_lines: true, encoding: 'utf8', })
+}
+
+const createChildrenForRoot = (parsedContent: Record<string, string>): TanaIntermediateNode[] | undefined => {
+    const keys = Object.keys(parsedContent)
+    const fields = keys.map(key => {
+        const newField = {
+            uid: idgenerator(),
+            name: key,
+            createdAt: new Date().getTime(),
+            editedAt: new Date().getTime(),
+            type: 'field',
+            todoState: undefined, 
+        }
+
+        const fieldChildren = [{
+            uid: idgenerator(),
+            name: parsedContent[key],
+            createdAt: new Date().getTime(),
+            editedAt: new Date().getTime(),
+            type: 'node',
+            todoState: undefined,
+        }]
+
+        return {...newField, children: fieldChildren}
+    }) as TanaIntermediateNode[]
+
+    return fields
+}
```

Comment:
All the fields are created for each key found in the `parsedContent`.  For [this pokemon notion](https://www.notion.so/8c117d7be79843798bf56c0b8fd522ae?v=f2861e5ff2ce4c76a560d1cf7fa600a3) a `parsedContent` object would look like this:

https://gist.github.com/theianjones/82188ec7fba89ead74ed85470c2e5d16